pyopenms_viz/_core.py

```
@@ -35,1071 +35,18 @@
-        ):
-            if optional in self.data.columns:
-                entries[optional.replace("_", " ")] = optional
-        # Create tooltips and custom hover data with backend specific formatting
-        tooltips, custom_hover_data = self._create_tooltips(
-            entries=entries, index=False
-        )
-        # Repeat data each time (since each peak is represented by three points in line plot)
-        custom_hover_data = repeat(custom_hover_data, 3, axis=0)
-
-        return tooltips, custom_hover_data
+class PeakMapPlot(BaseMSPlot, ABC):
```

Ruff (0.8.2)

Undefined name `BaseMSPlot`

(F821)